clojure/spec/gilded_rose/core_spec.clj

```diff
@@ -1,6 +1,9 @@
 (ns gilded-rose.core-spec
-(:require [speclj.core :refer :all]
+(:require [clojure.test :refer :all]
           [gilded-rose.core :refer [update-quality]]))
 
-(describe "gilded rose"
-          (it "TODO - write tests later"))
+(deftest update-quality
+  (testing "regular items"
+    (testing "before sell-by date"
+      (is (= [(gr/item "foo" 4 6)]
```

Comment:
What's `gr`?

Reply:
thats what i get for copying and pasting...